test/ninja/ninja_test_sconscripts/sconstruct_response_file

```diff
@@ -0,0 +1,15 @@
+SetOption('experimental','ninja')
+DefaultEnvironment(tools=[])
+
+vars = Variables()
+vars.Add('MLL', default='2048')
+
+env = Environment(variables=vars)
+env['MAXLINELENGTH'] = int(env['MLL'])
+env.Tool('ninja')
+
+env.Program(target='foo', source='foo.c', OBJSUFFIX=env['OBJSUFFIX'] + "1")
+
+env2 = env.Clone()
+env2.Append(CPPPATH='very/long/and/very/fake/path/for/testing')
```

Comment:
> __[Secret Scan]__ It may be dangerous to commit the AWS secret access key. ([view](https://sider.review/gh/repos/104670160/pulls/4133#issue-6868021))
>
> <details>
>
> AWS secret access key is an important credential that allows you to use AWS programmatically.
> Once this leaks, an attacker can access and operate your AWS resources.
>
> See also:
> * https://docs.aws.amazon.com/general/latest/gr/aws-access-keys-best-practices.html
>
> | Rule | Severity |
> | ---- | -------- |
> | `sider.secrets.aws.secret_access_key` | `warning` |
>
> You can [close this issue](https://sider.review/gh/repos/104670160/pulls/4133#issue-6868021) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

I closed this issue. Reason:
- When this is not an AWS secret access key.